docs/source/guides/getting_started/quick_start.rst

```diff
@@ -0,0 +1,76 @@
+Quick Start
+===========
+
+:ref:`Again <guides/getting_started/index:prerequisite knowledge>`, if the following
+serves all your needs, then you're probably better off with :ref:`one of the more
+user-friendly yt-dlp wrappers available <introduction:motivation>`. If you still want to
+get ``ytdl-sub`` up and running quickly and without understanding, then follow these
+instructions to the letter.
+
+#. Install using :ref:`the official Docker GUI image variant <guides/install/docker:gui
+   image>`.
+
+#. Update the paths for your media library:
+
+   Edit :ref:`the subscriptions file <guides/install/docker:configuration>`. Near the
+   top, under ``__preset__:`` and then ``overrides:``, update the values under the
+   ``*_directory:`` keys with the correct paths for your media library *as they appear
+   inside the container*.
+
+#. Select your media library software:
+
+   Change the ``Plex TV Show by Date:`` *key itself* to the preset for your media
+   library software. See the comment above for the available options.
+
+#. Select the genre:
+
+   Under the library software preset key from the previous step, change the ``=
+   Documentaries`` *key itself* to the genre for this subscription prefixed with ``=
+   ...``. You may specify multiple genres by separating them with ``|``, for example::
+
+     = Kids | = TV-Y:
+
+   When adding other subscriptions that have the same genre or genres, place them under
+   the same key.
+
+#. Update the subscription title and URL:
```

Comment:
micro-nit: `subscription title` -> `subscription name`

Reply:
I was actually wondering about exactly that when I wrote that. Thanks.